modules/rest-api/src/main/java/org/apache/ignite/internal/rest/api/configuration/ClusterConfigurationApi.java

```diff
@@ -86,20 +88,21 @@ public interface ClusterConfigurationApi {
      * @param path to represent a cluster configuration.
      * @return system configuration in HOCON format represented by given path.
      */
-    @Operation(operationId = "getClusterConfigurationByPath")
+    @Operation(operationId = "getClusterConfigurationByPath",
+            description = "Gets the configuration on the specific path. Configuration is in HOCON format")
     @ApiResponse(responseCode = "200",
             content = @Content(mediaType = MediaType.TEXT_PLAIN, schema = @Schema(type = "string")),
-            description = "Configuration represented by path")
-    @ApiResponse(responseCode = "500", description = "Internal error",
+            description = "Configuration of the cluster on the specified path.")
+    @ApiResponse(responseCode = "500", description = "Internal error.",
             content = @Content(mediaType = MediaType.PROBLEM_JSON, schema = @Schema(implementation = Problem.class)))
-    @ApiResponse(responseCode = "400", description = "Incorrect configuration",
+    @ApiResponse(responseCode = "400", description = "Incorrect configuration.",
             content = @Content(mediaType = MediaType.PROBLEM_JSON, schema = @Schema(implementation = Problem.class)))
-    @ApiResponse(responseCode = "404", description = "Configuration not found, it means that the cluster is not initialized",
+    @ApiResponse(responseCode = "404", description = "Configuration not found. Most likely, the cluster is not initialized.",
             content = @Content(mediaType = MediaType.PROBLEM_JSON, schema = @Schema(implementation = Problem.class)))
     @Produces({
             MediaType.TEXT_PLAIN,
             MediaType.PROBLEM_JSON
     })
     @Get("/{path}")
-    String getConfigurationByPath(@PathVariable("path") String path);
+    String getConfigurationByPath(@PathVariable("path") @Parameter(required = true, description = "Cluster address.")  String path);
```

Comment:
No, this is not the cluster address. This is the `path` that represents the configuration subtree. For example, given configuration:

```json
one: {
  two: {
    three: "value"
  }
}
```
Then the `path = one.two` will represent a configuration subtree:
```json
three: "value"
```